Update clustering-with-scikit-learn-in-python.md

anisa-hawes opened the pull request:
I am correcting two typing errors in this lesson.

which, Para. 422
in our data, Para. 750

Closes #2315 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [ ] ~~if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description~~
- [x] Add the appropriate "Label"
- [x] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [x] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

anisa-hawes committed:
Update clustering-with-scikit-learn-in-python.md

I am correcting two typing errors in this lesson.

which, Para. 422
in our data, Para. 750

svmelton commented:
@anisa-hawes I've got an issue open (#2314) about the broken link!

anisa-hawes commented:
Thank you, @svmelton! Yes, I saw that! Strangely, it seems like the Spanish translation of this QuickStart guide has been half-completed! On [this page](https://docs.github.com/es/get-started/quickstart/create-a-repo) for example, steps 1, 4, 5 and 6 have been translated into Spanish, while steps 2 and 3 remain in English. The banner concedes that translation into Spanish is still in progress.... but we don't know how long that might take...!

Do you think we should go ahead and temporarily update the link with the [alternative you identified](https://docs.github.com/es/get-started)? I note that translations into the other languages included in the dropdown menu (at upper right) are available.

svmelton commented:
> Thank you, @svmelton! Yes, I saw that! Strangely, it seems like the Spanish translation of this QuickStart guide has been half-completed! On [this page](https://docs.github.com/es/get-started/quickstart/create-a-repo) for example, steps 1, 4, 5 and 6 have been translated into Spanish, while steps 2 and 3 remain in English. The banner concedes that translation into Spanish is still in progress.... but we don't know how long that might take...!
> 
> Do you think we should go ahead and temporarily update the link with the [alternative you identified](https://docs.github.com/es/get-started)? I note that translations into the other languages included in the dropdown menu (at upper right) are available.

Yes, I think we can temporarily update the link until we can find something better. Thanks!

anisa-hawes committed:
Update clustering-with-scikit-learn-in-python.md

I am replacing links to data assets in our Jekyll repository with relative paths starting "/"

anisa-hawes committed:
Update clustering-with-scikit-learn-in-python.md

I am correcting URLs directing to assets in the Jekyll repository.
